### DISCUSSION TITLE
How to set default value for search filter

### DISCUSSION THREAD

**Discussion by anjanrs** [Laravel-Backpack/community-forum#477] [Q&A (Help)] [answered]
I want to set the default value for search filter when backpack list page is loaded, so that the result is filtered by the search value i sepcified.
How can i do it?

**Comment by o15a3d4l11s2** [comment, marked as answer]
As far as I know this is not possible inside the search field itself.

A possible workaround (if you are OK to have a filter instead of the search bar), is to create a text filter: https://backpackforlaravel.com/docs/5.x/crud-filters#text-1. Then you can change the menu link of your list from ` https://website.test/admin/product` to something like `https://website.test/admin/product?myfilter=test`. This way the admin will see the preset filter when they visit the list, but later on they would have to remove/update the filter value and not the search field if they want to change the results.

You can find some other possible solutions that require modifying the data table itself here: https://stackoverflow.com/questions/45202366/how-to-allow-get-request-to-search-laravel-backpack

**Comment by anjanrs** [comment]
Thank Dimitar for the reply, gave me some hint to make it work.
I create the customized the list view  by calling funciton below inside setupListOperation
$this->crud->setListView('your-view');

and adding the js code in the view
  <script>
      jQuery(document).ready(function($) {
          $('input[type=search]').val("{{Request::input('qsearch')}}");
          $('input[type=search]').trigger('keyup');
      });
  </script>
